inference-engine/src/vpu/myriad_plugin/myriad_executor.cpp

```diff
@@ -126,13 +124,6 @@ ncStatus_t MyriadExecutor::bootNextDevice(std::vector<DevicePtr> &devicePool, co
 
         if (it == availableDevicesDesc.end()) {
             IE_THROW() << "Myriad device: " << configDevName << " not found.";
-        } else {
```

Comment:
What about compiling with no device? Is it still supported that if there is no device compilation still successes, but an attempt to create an executable network will throw?

Reply:
In case when there is no device compilation successes. And no, there is no throw .